### DISCUSSION TITLE
Missing or wrong Channels, and searching for them and others.

### DISCUSSION THREAD

**Discussion by Omnipresent77** [iptv-org/community#1133] [Q&A]
West Puget Sound Area near seattle, WA state, USA
Using Kodi 21.1
Add-on PVR simple IPTV client

This is total wrong channels:

	#EXTINF:-1 tvg-id="KOMODT1.us" tvg-logo="https://i.imgur.com/8vXRrRe.jpg" group-title="Undefined",ABC 4 Seattle WA (KOMO-TV) (720p)
	https://content.uplynk.com/2c88dfe19e1447e6a6aa27e8e143a140.m3u8

		Is NOT Komo 4 ABC from Seattle, WA state, USA. It shows a looped 9 seconds of the Komo TV logo is it.

	#EXTINF:-1 tvg-id="KINGDT1.us" tvg-logo="https://i.imgur.com/fTRLwhP.png" group-title="Undefined",NBC 5 Seattle WA (KING-DT1) (1080p)
	https://livevideo.tegnadigital.com/king/v1/master/f9c1bf9ffd6ac86b6173a7c169ff6e3f4efbd693/KING/elvs/live.m3u8

		Sometimes news is in synce but other wise this is a incorrect channel (basically a loop of the news, no actuall shows are in sync). NOT NBC King 5 Seattle, WA USA.
	
	#EXTINF:-1 tvg-id="KIRODT1.us" tvg-logo="https://i.imgur.com/suvT0TN.png" group-title="Undefined",KIRO-DT1 (1080p)
	https://cdn-ue1-prod.tsv2.amagi.tv/linear/amg00327-coxmediagroup-kirobreaking-ono/playlist.m3u8
	
		Sometimes news is in synce but other wise this is a incorrect channel (basically a loop of the news, no actuall shows are in sync). Not CBS Kiro 7 Seattle WA USA.
	
	#EXTINF:-1 tvg-id="HistoryHD.si" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/0/01/History_%282021%29.svg/512px-History_%282021%29.svg.png" group-title="Documentary",History Channel (1080p)
	https://streamer12.xploretv.si/__cl/cg:prod/__c/A1_SI_HISTORYHD_ott/__op/dash-default/__dci/__f/index.m3u8?admin=xploreTv_test_user&redirect=true
	
		Just says failed to play error ( MANY channels say this also.)

Working channels. I wanted and seem correct.

	#EXTINF:-1 tvg-id="BBCAmericaEast.us" tvg-logo="https://i.imgur.com/pTiiPz0.png" group-title="Entertainment",BBC America East
	http://fl2.moveonjoy.com/BBC_AMERICA/index.m3u8
	
		BBC America East
	
	#EXTINF:-1 tvg-id="SyfyEast.us" tvg-logo="https://i.imgur.com/4fMWhUE.png" group-title="Undefined",Syfy East
	http://fl2.moveonjoy.com/SYFY/index.m3u8
	
		SyFy Channel East
	
Channels I can not find at all:

Discovery:
tvg-logo="https://logodownload.org/wp-content/uploads/2017/04/discovery-channel-logo-0.png"

Science:
tvg-logo=blue square with sides extended like a cube with the letters SCI on it.

Travel: 
tvg-logo=just a TRVL  in an odd font.

**Comment by Omnipresent77** [comment]
So after so looking although those BBC, and SYFY channels work. i would love to have them 3 hours shifted for the west coast. Is that what time shifting a channel would do maybe??

**Comment by BellezaEmporium** [comment]
Discovery, Science & Travel are DMCAd.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-report-a-broken-stream
